src/DesktopVirtualization/docs/New-AzWvdHostPool.md

```diff
@@ -103,6 +106,69 @@ This command creates a Windows Virtual Desktop HostPool in a Resource Group.
 
 ## PARAMETERS
 
+### -AgentUpdateMaintenanceWindow
```

Comment:
These new parameters could really use an example

Reply:
I  updated the example to include the new agent update parameters